manifests/rhoai/shared/odhdashboardconfig/odhdashboardconfig.yaml

```diff
@@ -1,41 +1,14 @@
-# ODHDashboardConfig to enable the notebook-controller feature and add notebook sizes
+# Read the README before modifying
 apiVersion: opendatahub.io/v1alpha
 kind: OdhDashboardConfig
 metadata:
-  creationTimestamp: null
   name: odh-dashboard-config
+  annotations:
+    opendatahub.io/managed: "false"
```

Comment:
Moving away from custom code in the Operator -- this has no impact on the way this works, just clearer.
See https://github.com/opendatahub-io/architecture-decision-records/pull/93 for more information

Platform tracked issue to remove the custom code: https://issues.redhat.com/browse/RHOAIENG-23603